src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -69,31 +71,36 @@
 
 (defn token-value-drawer
   [token watch-only? entry-point]
-  (let [token-symbol      (:token token)
-        token-data        (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
-                                          token-symbol]))
-        selected-account? (rf/sub [:wallet/current-viewing-account-address])
-        token-owners      (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
-        send-params       (if selected-account?
-                            {:token       token-data
-                             :stack-id    :screen/wallet.accounts
-                             :start-flow? true
-                             :owners      token-owners}
-                            {:token-symbol token-symbol
-                             :stack-id     :wallet-stack
-                             :start-flow?  true
-                             :owners       token-owners})
-        bridge-params     (if selected-account?
-                            {:token            token-data
-                             :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                             :stack-id         :screen/wallet.accounts
-                             :start-flow?      true
-                             :owners           token-owners}
-                            {:token-symbol     token-symbol
-                             :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                             :stack-id         :wallet-stack
-                             :start-flow?      true
-                             :owners           token-owners})]
+  (let [token-symbol           (:token token)
+        token-data             (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
+                                               token-symbol]))
+        fiat-unformatted-value (get-in token [:values :fiat-unformatted-value])
+        has-balance?           (money/greater-than fiat-unformatted-value (money/bignumber "0"))
```

Comment:
main change here is to get the unformatted balance and check it's greater than 0? 
If it's not then disable the send and bridge options - and only in the account page view